rust/noosphere/include/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module Noosphere {
+    header "noosphere.h"
+    export *
+}
```

Comment:
Is this for swift? How is this used?

Reply:
Yes, this is for XCode packaging ceremony. It is required for... reasons I don't fully understand, TBH. You need it to exist at the `.xcframework` generation step.